proposals/lip-replace_static_fee_system_by_dynamic_fee_system.md

```diff
@@ -0,0 +1,358 @@
+    - Voting transaction (with 2nd passphrase): 2334 bytes* 10<sup>-7</sup> LSK/byte = 0.0002334 LSK
+    - Minimum fee of a full block<sup>[1]</sup>: 15 KB* 10<sup>-7</sup> LSK/byte = 0.0015 LSK
+
+    With this minimum fee, and with a quiet network, a malicious user can issue **100000 transactions** spending just a bit more than 1 LSK. These transactions will cause around of **15 MB of spam** in the network. Assuming 1 LSK < 40$, the minimum fee for a full vote transaction will be below 0.01$ cost, which is cheap enough to encourage a dynamic and vivid voting system.
+
+3.  **Two orders of magnitude greater (10<sup>-6</sup> LSK/byte)**:
+    - Balance transaction: 125 bytes* 10<sup>-6</sup> LSK/byte = 1.25 *10<sup>-4</sup> LSK (1.82 * 10<sup>-4</sup> LSK with 2nd passphrase)
+    - Voting transaction (with 2nd passphrase): 2334 bytes* 10<sup>-6</sup> LSK/byte = 0.002334 LSK
+    - Minimum fee of a full block<sup>[1]</sup>: 15 KB* 10<sup>-6</sup> LSK/byte = 0.015 LSK
+
+    With this minimum fee, and with a quiet network, a malicious user can issue **10000 transactions** spending just a bit more than 1 LSK. These transactions will cause around of **1.5 MB of spam** in the network. Assuming 1 LSK < 40$, the minimum fee for a full vote transaction will be below 0.1$ cost, which is cheap enough to encourage a dynamic and vivid voting system.
+
+4.  **Three Orders of magnitude greater (10<sup>-5</sup> LSK/byte)**:
+    - Balance transaction: 125 bytes* 10<sup>-5</sup> LSK/byte = 0.00125 LSK (0.00182 LSK with 2nd passphrase)
+    - Voting transaction (with 2nd passphrase): 2334 bytes* 10<sup>-5</sup> LSK/byte = 0.02334 LSK
+    - Minimum fee of a full block<sup>[1]</sup>: 15 KB* 10<sup>-5</sup> LSK/byte= 0.15 LSK
+
+    With this minimum fee, and with a quiet network, a malicious user can issue **1000 transactions** spending just a bit more than 1 LSK. These transactions will cause around of **150 KB of spam** in the network.  Assuming 1 LSK < 40$, the minimum fee for a full vote transaction will be below 1$, which will encourage a dynamic voting situation in most of the cases.
+
+<sup>[1]</sup>We are assuming that a full block has a size of 15 KB (15000 bytes).  Also, we are assuming that they do not include any transaction of types 2 or 5. In those cases, the fee per block will be higher due to the `trs.nameFee` constant included in the fee of these transactions.
+
+### B: Namespace fee implications
+
+Minimum fee, `trs.minFee`, for transaction types 2 and 5 for different values of `trs.nameFee` for the current protocol reference implementation:
+
+1. **`trs.nameFee` = 0:**
+
+**Tx type/ Min fee** | **1*10<sup>-7</sup> LSK/byte** | **5*10<sup>-7</sup> LSK/byte** | **1*10<sup>-6</sup> LSK/byte** | **5*10<sup>-6</sup> LSK/byte** | **1*10<sup>-5</sup> LSK/byte**
+--- | --- | --- | --- | --- | --- 
+**Delegate reg (145 bytes)** | 1.45*10<sup>-5</sup> LSK | 7.25*10<sup>-5</sup> LSK | 1.45*10<sup>-4</sup> LSK | 7.25*10<sup>-4</sup> LSK | 0.0013 LSK
```

Comment:
the last value must be 0.00145 LSK.
Also for the following tables, the last entry of the first row has to be updated.

Reply:
good catch! updated